main/src/library/GetOpt.flix

```diff
@@ -1,418 +1,418 @@
-/*

```

Comment:
Why does this entire file show up as changed? What changed??

Reply:
The file mode changed (100755 → 100644). I guess it was executable before?

Reply:
Looks like this file was CRLF. When I piped everything through a script, it changed it to LF.